content/patterns/table/examples/sortable-table.html

```diff
@@ -65,7 +65,7 @@ <h2 id="ex_label">Example</h2>
           <table class="sortable">
             <caption>
               Students currently enrolled in WAI-ARIA 101
-              <span class="sr-only">, column headers with buttons are sortable.</span>
+              <span class="sr-only"> (column headers with buttons are sortable).</span>
```

Comment:
Very strange ... when I look at the preview, the space preceding the parenthesis is not there. What would be causing it to get trimmed? Something about the class? Should we do something like include a `<br>` prior to or inside the `<span>`. I don't understand what is happening. This should be simple.

Reply:
hmm, I am seeing a space in VoiceOver output.... what screenreader are you using? Maybe we can discuss in the meeting tomorrow. Could also try adding `&nbsp;`

<img width="1322" alt="Screenshot 2024-08-12 at 8 34 31 PM" src="https://github.com/user-attachments/assets/f6469238-58bf-44d6-8c14-ecc08ee77576">

Reply:
```suggestion
              <span class="sr-only">&puncsp;(column headers with buttons are sortable).</span>
```

Heya @ariellalgilmore, the breaking space entity I was trying to remember is [`&puncsp;`](https://html.spec.whatwg.org/multipage/named-characters.html#:~:text=%CF%88-,puncsp%3B,-U%2B02008) — it might be worth a shot. It looks like there are other white space entities that might work well too: `&ensp;`, `&thinsp;`, etc.

I also found the related w3c/accname#225, which cross-references a CSS proposal named [`white-space-trim`](https://www.w3.org/TR/css-text-4/#propdef-white-space-trim).

cc: @mcking65

Reply:
Thank you @adampage! I am having trouble testing this on my devices. I am not the strongest JAWS user, maybe we can sync on how I can test this locally one time. I committed the suggestion in the meantime though, so it can be tested in the deploy preview.